tests/src/test/scala/Demo.scala

```diff
@@ -63,6 +75,10 @@ final class NestedParent() {
   case class NestedChild(name: String)
 }
 
+object ObjectNestedParent {
```

Comment:
Gonna need a test case where this object is in a class - in that case we shouldn't be changing the `toString` at all (I want to keep consistent behavior for things nested in classes for now)

Reply:
Just added that in https://github.com/kubukoz/better-tostring/pull/27/commits/7ac685015a1d49bc5a9acbbe773d0eeadd42f7e6